README.rst

```diff
@@ -659,6 +659,11 @@ Choose from the list of available fixers:
                         $strict param. Warning! This
                         could change code behavior.
 
+* **useless_return** [contrib]
```

Comment:
So standard procedure - send a PR to fix them.
Looking on the scale I think it will be merged, so Symfony level indeed.